netci.groovy

```diff
@@ -18,18 +18,28 @@ def msbuildTypeMap = [
 
 // convert `machine` parameter to OS component of PR task name
 def machineTypeToOSTagMap = [
-    'Windows_NT': 'Windows',        // Windows Server 2012 R2, equivalent to Windows 8.1 (aka Blue)
+    'Windows 7': 'Windows 7',       // Windows Server 2008 R2                     ~= Windows 7
+    'Windows_NT': 'Windows',        // 'latest-or-auto' -> Windows Server 2012 R2 ~= Windows 8.1 aka Blue
+                                    // 'latest-dev15-5' -> Windows Server 2016    ~= Windows 10 RS2/RS3, with Dev15.5
```

Comment:
@mmitche do you know the exact build number here? In our case it may matter exactly which build it is. /cc @jackhorton

Reply:
For context, I would like to try it out locally to see if we can use the ICU on the system or not. Additionally, I would be interested in knowing which Windows SDK it will have (the RS2 SDK has one set of header files for ICU, while the RS3+ SDK has a different header. I believe the DLLs are compatible, but the headers weren't because they switched from wchar_t/uint16_t to char16_t). /cc @jefgen

Reply:
Right, the DLLs should indeed be binary compatible, though the headers are different between RS2 and RS3+. In other words, if you use the RS2 headers and build on RS2, then your resulting binary should run on RS3. (cc: @Microsoft/target-dev )